addons/web/static/src/scss/web_calendar.scss

```diff
@@ -542,6 +632,10 @@ $o-cw-filter-avatar-size: 20px;
         line-height: 1;
     }
 
+    .card-footer {
```

Comment:
remaining of the file is not linked to year mode
this is a "re-style" of the popup in the other modes

Reply:
I would maybe squash those two commits (basically, you rework the style of mini calendars)